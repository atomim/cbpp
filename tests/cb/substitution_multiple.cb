#define SQUARESUM(a,b,c) (a*a + b + c)

Print "sumstuff= " + SQUARESUM(1,2,3)
WaitKey
