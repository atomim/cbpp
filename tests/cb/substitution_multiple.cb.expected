// #define SQUARESUM(a,b,c) (a*a + b + c)

Print "sumstuff= " + (1*1 + 2 + 3)
WaitKey
